mailgun/route_handlers.py

```diff
@@ -79,13 +84,18 @@ def handle_mailing_list_email_route(request):
         subject = "Undeliverable mail"
         ml.send_mail(sender, subject, html=content)
     else:
-        # Do not send to the sender
+        # Do not send to the sender and also check if it is a reply-all and do not send to original sender to avoid
+        # duplicate being sent as the email client would have already sent it per the Reply-To param set in header
         try:
             member_addresses.remove(sender)
+            if in_reply_to:
+                member_addresses.remove(original_to_address)
```

Comment:
Just a warning that this may be a little fragile...you're assuming they don't rearrange the to/cc fields.  Ideally what we want to do is generate a list of all to/cc addresses, and if any of those are in the list, don't send to them.